packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -93,7 +95,35 @@ - (void)alertMessage:(NSString *)message
                                                                                    completion:nil];
 }
 
-- (void)getAvailableBiometrics:(FlutterResult)result {
+- (void)deviceSupportsBiometrics:(FlutterResult)result {
+  LAContext *context = self.createAuthContext;
+  NSError *authError = nil;
+  // Check if authentication with biometrics is possible.
+  if ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics
+                           error:&authError]) {
+    if (authError == nil) {
+      result(@YES);
+      return;
+    }
+  }
+  // If not, check if it is because no biometrics are enrolled (but still present).
+  if (authError != nil) {
+    if (@available(iOS 11.0.1, *)) {
```

Comment:
Why is this 11.0.1, rather than 11.0?

Reply:
It was a copy-paste from some [existing code bit further down](https://github.com/flutter/plugins/blob/8e887013a4d48ffc2e6f5a19766946566f0790f0/packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m#L133=), however I can't find a reason for that one to be 11.0.1 either, so I've changed both cases to just check for 11.